bitnami/postgresql-ha/templates/ldap-secrets.yaml

```diff
@@ -4,6 +4,12 @@ kind: Secret
 metadata:
   name: {{ include "postgresql-ha.ldap" . }}
   labels: {{- include "postgresql-ha.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "postgresql-ha.tplValue" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "postgresql-ha.tplValue" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
```

Comment:
Can't we add the common chart as a dependency for postgresql-ha without creating a backward incompatibility? In that way, we can use the `common.tplvalue` funtion